### PR TITLE
Added nav bar to application view layout

### PR DESCRIPTION
**Pull request by tofuandeve**
Created a nav bar for see all merchants, login, logout buttons

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,8 +8,29 @@
...
+
+        <a class="navbar-brand" href="/">Bandit</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav mr-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="/products"> All Merchants</a>
```
Comment (mdove92) [inline app/views/layouts/application.html.erb]:
```suggestion
              <a class="nav-link" href="/"> All Merchants</a>
```

**Review by emaust — COMMENTED** [review]
Change href to link_to

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -8,8 +8,29 @@
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
-
   <body>
+    <header>
+      <nav class="navbar navbar-expand-lg navbar-light fixed-top navbar-custom">
+
+        <a class="navbar-brand" href="/">Bandit</a>
```
Comment (mdove92) [inline app/views/layouts/application.html.erb]: should be link_to

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -8,8 +8,29 @@
...
+
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav mr-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="/products"> All Merchants</a>
+            </li>
+          </ul>
+
+          <ul class="navbar-nav navbar-right">
+            <li class="nav-item"><a class="nav-link" href="/">Login</a></li>
```
Comment (mdove92) [inline app/views/layouts/application.html.erb]: make link_to

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -8,8 +8,29 @@
...
+            <li class="nav-item"><a class="nav-link" href="/">Logout</a></li>
```
Comment (mdove92) [inline app/views/layouts/application.html.erb]: make link_to

**Review by michaela260 — APPROVED** [review]

**Review by mdove92 — CHANGES_REQUESTED** [review]
Great work! Change the hrefs to link_tos

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -12,25 +12,36 @@
...
+            <% else %>
+              <li class="nav-item app-header__nav_item">
+                <%= link_to "Login with GitHub", root_path, class: "btn btn-primary" %>
+              </li>
+            <% end %>  
           </ul>
         </div>
       </nav>
     </header>
+
```
Comment (emaust) [inline app/views/layouts/application.html.erb:44]: starting body tag missing

File: config/routes.rb [hunk 6]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root to: 'homepages#index'
```
Comment (emaust) [inline config/routes.rb]: this should be homepage#index (remove s)

**Review by emaust — CHANGES_REQUESTED** [review]

**Review by mdove92 — APPROVED** [review]

File: config/routes.rb (hunk 6 above)
Comment (tofuandeve) [inline config/routes.rb:3]: @emaust Homepage is a controller, shouldn't it always be plural in the route to action?

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (tofuandeve) [reply app/views/layouts/application.html.erb:44]: @emaust it's on line 46

Comment (tofuandeve) [reply app/views/layouts/application.html.erb:44]: it's on line 46

File: config/routes.rb (hunk 6 above)
Comment (tofuandeve) [reply config/routes.rb]: homepage is a controller, shouldn't it always be plural for route to controller action?

**Review by emaust — APPROVED** [review]

**Pull request merged**